common/taskana-common/src/main/resources/sql/db2/taskana_schema_update_5.10.0_to_6.2.0_db2.sql

```diff
@@ -0,0 +1,11 @@
+-- this script updates the TASKANA database schema from version 5.10.0 to version 6.2.0.
```

Comment:
Please add "SET SCHEMA %schemaName%;" like in the most db2 update scripts

Reply:
fixed